dbms/src/TiDB/Schema/TiDBSchemaSyncer.h

```diff
@@ -144,64 +145,82 @@ struct TiDBSchemaSyncer : public SchemaSyncer
         return it->second;
     }
 
-    bool tryLoadSchemaDiffs(Getter & getter, Int64 version, Context & context)
+    Int64 tryLoadSchemaDiffs(Getter & getter, Int64 lastest_version, Context & context)
```

Comment:
1. Use `latest_version` instead of `lastest_version`
2. Add some comments about the meaning of the return value